docs/UserGuide.md

```diff
@@ -152,6 +170,26 @@ e.g. the `NAME` parameter has a delimiter `n/` and the `AMENITY` parameter has a
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 </div>
 
+Here is a list of common parameters used.
+
+Parameters | Features that support these parameters
+--------|------------------
+Name `n/` | `add`, `edit`, `fav`, `unfav`, `delete`
+Spot `spot/` | `edit`
+Address `a/` | `add`, `edit`
+Operating Hours `o/` | `add`, `edit`
+Studied Hours `hr/` | `add`, `log`
+Rating `r/` | `add`, `edit`, `list`
+Amenity `m/` | `add`, `edit`, `list`
+Tag `t/` | `add`, `edit`, `list`
+Remove tag `rt/` | `edit`
+Remove amenity `rm/` | `edit`
+Alias `al/` | `alias`, `unalias`
+Command `cmd` | `alias`, `unalias`
+
+<div markdown="span" class="alert alert-primary">:information_source: **Note:**
+Flags may differ in meaning depending on the command word used. 
+</div>
```

Comment:
Should be '_Parameters_ may differ in meaning...'

Reply:
Oops, this part should only be for flags as there were different meanings for `-r`